Applications/DataExplorer/VtkVis/VtkCompositeImageToPointCloudFilter.cpp

```diff
@@ -0,0 +1,103 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2018, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#include "VtkCompositeImageToPointCloudFilter.h"
+
+#include <vtkPointData.h>
+#include <vtkSmartPointer.h>
+
+#include <QMap>
+#include <QString>
+#include <QVariant>
+
+#include "VtkImageDataToPointCloudFilter.h"
+
+VtkCompositeImageToPointCloudFilter::VtkCompositeImageToPointCloudFilter(vtkAlgorithm* inputAlgorithm)
+    : VtkCompositeFilter(inputAlgorithm)
+{
+    init();
+}
+
+VtkCompositeImageToPointCloudFilter::~VtkCompositeImageToPointCloudFilter() = default;
```

Comment:
This can be moved into the header, so the reader immediately sees, that the implementation is the default one...

Reply:
How do it match this with ```~VtkImageDataToPointCloudFilter() override;``` specified in the header file?